### PR TITLE
fix(SubscriberHistory): change component name to MasterSubscriber

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Change subscriber history to master subscriber

### CODE REVIEW

File: src/components/Reports/SubscriberHistory/SubscriberHistory.test.js [hunk 1]
```diff
@@ -1,23 +1,219 @@
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, wait } from '@testing-library/react';
```
Comment (cbernat) [inline src/components/Reports/SubscriberHistory/SubscriberHistory.test.js]: Why did you add this new `wait` here? it was not here before.

Comment (cbernat) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.test.js]: Forget this comment, this was marked as change in all files view, but in this commit it has no changes.

**Comment by cbernat** [comment]
Seems ok, thanks for the small commits, it makes it a lot easier to understand. 🚀

File: src/components/Reports/SubscriberHistory/SubscriberHistory.test.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/SubscriberHistory/SubscriberHistory.test.js]: because in the old file before rename dont have and now yes in the new file
